controllers/message-utils.js

```diff
@@ -1,20 +1,97 @@
-var db = require('../db');
-
-var updateCouchDB = function(record_id, body, callback) {
-  db.medic.updateWithHandler(
-    'medic',
-    'update_message_task',
-    record_id,
-    body,
-    callback
-  );
-};
+const _ = require('underscore'),
+      db = require('../db');
 
-var getTaskMessages = function(options, callback) {
+const getTaskMessages = function(options, callback) {
   db.medic.view('medic', 'tasks_messages', options, callback);
 };
 
+// copied from kujua-utils
+// TODO: get rid of this copy once Milan's refactor is complete
```

Comment:
If there's a ticket number for "milan's refactor" it would be great to include it with this comment